src/website/src/app/documentation/demos/stepper/angular-stepper-reactive.demo.html

```diff
@@ -0,0 +1,71 @@
+<h4>Angular - Reactive Forms</h4>
+
+<button (click)="form.reset()" class="btn">Reset Stepper</button>
+
+<form clrStepper [formGroup]="form" (ngSubmit)="submit()">
+  <clr-stepper-panel formGroupName="name">
+    <clr-step-title>Legal Name</clr-step-title>
+    <clr-step-description
```

Comment:
nit: I know it's just an example, but can we have some description that's more related to the actual step we're showing? It's the same text for all the steps below too. And it's visible both in the example component and the example source below it.

Reply:
Ah you mean make it the same as the screenshot at the top?

Reply:
I mean the following fields:
```
user             The full license information can be found in LICENSE in the root directory.
password    The full license information can be found in LICENSE in the root directory.
whatever     The full license information can be found in LICENSE in the root directory.
```